pkg/cri/resource-manager/cache/cache.go

```diff
@@ -490,6 +490,12 @@ type Cache interface {
 const (
 	// CacheVersion is the running version of the cache.
 	CacheVersion = "1"
+	// cacheFilePermissions is the unix permissions for the cache file
+	cacheFilePermissions = os.FileMode(0640)
+	// cacheDirPermissions is the unix permissions for the directory of the cache file
+	cacheDirPermissions = os.FileMode(0710)
```

Comment:
Here the original permission was stricter... but maybe the group access is a good thing if we move later running the service as a user.

Reply:
Yes, that's deliberate. Maybe the commit message is a bit misleading.